### PR TITLE
Fix sample projects

### PR DESCRIPTION
**Pull request by const-cloudinary**
Make them compatible with the latest Rails

### CODE REVIEW

File: samples/basic-rails/Gemfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.21'
+gem 'rails'
```
Comment (amcaplan) [inline samples/basic-rails/Gemfile:3]: We should probably tie to a version (maybe latest)

File: samples/photo_album/Gemfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'rails', '3.2.21'
-gem 'sqlite3', '~>1.3.5'
+gem 'rails'
```
Comment (amcaplan) [inline samples/photo_album/Gemfile:3]: same

File: samples/photo_album/config/application.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 require "active_record/railtie"
 require "action_controller/railtie"
 require "action_mailer/railtie"
-require "active_resource/railtie"
+# require "active_resource/railtie"
```
Comment (amcaplan) [inline samples/photo_album/config/application.rb:7]: ?

Comment (const-cloudinary) [reply samples/photo_album/config/application.rb]: This module was removed from rails long ago and our sample project does not use it.

**Pull request merged**